index.js

```diff
@@ -0,0 +1,88 @@
+// hamburger menu
+// document.querySelector(".hamburger").addEventListener("click", () => {
+//     document.querySelector("nav").classList.toggle("hide");
+// })
+
+// document.querySelector(".nav-close-btn").addEventListener("click", () => {
+//     document.querySelector("nav").classList.toggle("hide");
+// })
+
```

Comment:
still remove commented code!!

Reply:
working on that functionality, so commented.